packages/core/src/utils/index.js

```diff
@@ -9,6 +9,14 @@ const {
   markdownFileExts,
 } = require('../constants');
 
+const htmlUnescapedMapping = {
```

Comment:
how about something like https://www.npmjs.com/package/he?

might become a little tedious to maintain the list

Reply:
This mapping is taken from `markdown-it` library, just reversed. I did this as in our codebase we use `markdown-it/utils` function `escapeHtml` for a lot of purposes, but that package didn't expose unescaping methods for HTML, so I took it upon myself to add a complementary function. Looks like it's going to be only that 5 entries though, as the fact that this entry only aims to reverse the `escapeHtml` function and that function only escapes those 5 entries.

Though if you feel we can add a new package to make everything concise I'm also okay with it.

Reply:
ahh, thanks for clarifying!

In that case let's stick with this then, we might not want to unescape things markdown-it did not escape as well.

On the same lines, should we put this somewhere in the lib/markdown-it folder? Also a comment to document why / where the mappings came from may help 🙂

Reply:
:+1: I agree. This is more related to operations involving markdown-it as well so it makes sense to put it in the lib/markdown-it folder. Though I can't actually expose the new function through the `markdownIt` object in `lib/markdown-it/index.js`, and I can't disturb the exports there unless I replace all imports in the project, so I will just make it available for manual import through `lib/markdown-it/utils`.

It's the best middle ground I have, but I'm afraid it will somewhat be confusing as now we have two different modules for utilities on markdown-it, one from the library itself usually referred to as `md.utils`, the other is the new `utils` module. I might put an explanation on the top of our `utils` module that this is a separate extension of the one from the library.